### ISSUE TITLE
[Android] [SNES9X] Visual hiccup during battle transitions for Chrono Trigger.

### ISSUE THREAD

**Issue by Gurk232** [libretro/libretro-meta#83]
During battle transitions in Chrono Trigger, there's a visual glitch where the screen seems to jilt upward very slightly, this ruins the normally smooth transition.

This issue does not occur in Snes9x 2010, only in regular Snes9x.

**Comment by sergiobenrocha2** [comment]
Is it really a bug? Maybe your phone is weak / don't have enough power to run smoothly Snes9x

**Comment by Gurk232** [comment]
Nah it runs fine, I'll attempt it on a more powerful device if you really want me to though.

**Comment by Gurk232** [comment]
Have you tried replicating the bug yourself just to make sure?

**Comment by inactive123** [comment]
Yes this is a known issue, you can hack around this which I did in snes9x 2010, I guess this isn't being done in the mainline snes9x core right now.

**Comment by Gurk232** [comment]
Thanks for confirming, Does anyone know what causes it just out of curiosity?

**Comment by Awakened0** [comment]
This happens on Windows too, so maybe the Android label isn't needed.

**Comment by Awakened0** [comment]
I kind of remember this bug now. Isn't it related to the [overscan padding issue](https://github.com/libretro/libretro-meta/issues/6)? If you turn off crop overscan it doesn't happen.

**Comment by Fergdog** [comment]
It only happens when crop overscan is set to AUTO, it doesn't happen when it's set to ON or OFF. Something about the battle transition causes issues with AUTO.

**Comment by ghost** [comment]
this CT issue should not be a problem now in latest commits of snes9x
